sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -74,29 +81,89 @@
     </item>
     <item>
       <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <Description>Time at which the most recent data was read (TAI unix seconds).</Description>
       <IDL_Type>double</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>direction</EFDB_Name>
-      <Description>Wind direction: 0 = north, 90 = east.</Description>
+      <Description>Median (mean for some sensors) wind direction measured over a period on the order of a minute: 0 = north, 90 = east.</Description>
```

Comment:
We may have discussed this already and if we did I apologize for not having remembered. Why median in some and mean in other cases? Is it possible or even favorable to always report the median or always the mean instead? Same question for the other items that report either median or mean below.

Reply:
Median is a "best effort". We can compute it for sonic anemometers, but for weather stations we are stuck with whatever they report -- presumably mean.